### PR TITLE
Start working on network automation docs

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

Feedback much appreciated

Ping @whiteinge

### CODE REVIEW

File: doc/topics/network_automation/index.rst [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+- :mod:`NAPALM Helpers <salt.modules.napalm>` - Generic helpers for
+  NAPALM-related operations. For example, the
+  :mod:`Compliance report <salt.modules.napalm.compliance_report>` function
+  can be used inside the state modules to compare the expected and the
+  existing configuration.
+
+Getting started
+###############
+
+Install NAPALAM - follow the notes_ and check the platform-specific dependencies_.
```
Comment (cachedout) [inline doc/topics/network_automation/index.rst]: `NAPALM`

**Review by cachedout — CHANGES_REQUESTED** [review]
Really great! Just one small typo to fix.

File: doc/topics/network_automation/index.rst (hunk 1 above)
Comment (mirceaulinic) [inline doc/topics/network_automation/index.rst]: Nice catch!

**Comment by whiteinge** [comment]
This builds good locally. IMO this should be added to the top-level TOC:

```diff
diff --git a/doc/contents.rst b/doc/contents.rst
index cb1e3e52a2..3dea7e04c4 100644
--- a/doc/contents.rst
+++ b/doc/contents.rst
@@ -17,6 +17,7 @@ Salt Table of Contents
     topics/orchestrate/index
     topics/ssh/index
     topics/cloud/index
+    topics/network_automation/index
     topics/proxyminion/index
     topics/virt/index
     ref/cli/index
```

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
Thank you @mirceaulinic!

**Comment by mirceaulinic** [comment]
@gtmanfred @vnitinv @tonybaloney would you be able to continue adding bits under this topic?

**Comment by vnitinv** [comment]
@mirceaulinic Will add for junos. Thanks

**Comment by vnitinv** [comment]
@rajvidhimar Can you please take care of this.

**Comment by tonybaloney** [comment]
@vnitinv it's merged

**Pull request merged**